cpp/src/arrow/type.h

```diff
@@ -1652,7 +1652,7 @@ class ARROW_EXPORT Schema : public detail::Fingerprintable,
   /// \brief Return the number of fields (columns) in the schema
   int num_fields() const;
 
-  /// Return the ith schema element. Does not boundscheck
+  /// Return the with schema element. Does not boundscheck
```

Comment:
This is incorrect- it was intended to be `ith` as in the element at position `i`

Reply:
@bkietz Indeed, updated.

Reply:
Perhaps "i-th" to make it less ambiguous?